apps/language_models/src/pipelines/SharkLLMBase.py

```diff
@@ -3,11 +3,16 @@
 
 class SharkLLMBase(ABC):
     def __init__(
-        self, model_name, hf_model_path=None, max_num_tokens=512
+        self,
+        model_name,
+        hf_model_path=None,
+        max_num_tokens=512,
```

Comment:
Why hard code this ?

Reply:
Set to false by default for minimal implementations, but I don't think the changes to LLMBase are necessary after all.